Assets/MixedRealityToolkit.Services/DiagnosticsSystem/MixedRealityToolkitVisualProfiler.cs

```diff
@@ -259,6 +270,10 @@ private void Start()
             Reset();
             BuildWindow();
             BuildFrameRateStrings();
+
+#if WINDOWS_UWP
+            appCapture = AppCapture.GetForCurrentView();
```

Comment:
There are other locations in our code that marshal onto the UI thread (i.e. dispatch to UI thread) in order to access GetForCurrentView APIs - are those just... wrong (i.e. they're also called I think from similar Unity functions)?

Reply:
So...yeah, I was expecting this call to be like those as well, but from testing, this call works just fine on the main thread. The others definitely needed to be on the UI thread though.

Reply:
I know this is already merged, but I just wanted to add another thought here. Is it possible the call could come from another thread (not Unity UI or Main)? I'm thinking, for example, if a customer turned on or off the profiler when an Azure Spatial Anchor is located. That happens on a worker Task thread. This may work just fine, but I wanted to call it out.

Reply:
Oh, good call! I'll double check this